### PR TITLE
Machinery Resprite

### PR DESCRIPTION
**Pull request by alsoandanswer**
Sorts objects.dmi and stationobjs.dmi

Ports machinery sprites from /tg/ and bay. Updates old codersprites.

For Danse and Desven.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/machinery/autolathe/autolathe.dm [hunk 1]
```diff
@@ -335,7 +344,16 @@
 	else if(fill_status[FILL_INCOMPLETELY])
 		to_chat(user, SPAN_NOTICE("You fill \the [src] with [english_list(fill_status[FILL_INCOMPLETELY])] \the [eating]."))
 
-	flick("autolathe_o", src) // Plays metal insertion animation. Work out a good way to work out a fitting animation. ~Z
+	var/image/I = image(icon, "[initial(icon_state)]_mat")
+	var/mat_colour
+	if(eating.color)
+		mat_colour = eating.color
+	//	else
+	//	 mat_colour = eating.matter[material].icon_color
```
Comment (NonQueueingMatt) [inline code/game/machinery/autolathe/autolathe.dm:352]: Commented code.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
The weapons analyzer places an overlay of the weapon over itself, which is now unaligned with the new sprite:
immagine

**Comment by ScheveningenZonn** [comment]
image

The images on the right for the RND machinery appear to lack design consistency between various pieces of machinery (silver-colored to deep carbon black to silver-colored again). The machinery as we have now are Aurora-unique. The RND machinery in your proposal very clearly have TG's spriting design philosophy in mind and not one of an aesthetic unique to Aurora. I see no reason to go backward to what we had before.

The current pipe dispenser and shield generators are very ugly and would be welcome to have new sprites, but I think you are making a mistake overwriting the RND machinery.

File: html/changelogs/wezzy_machinery_scenery.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - imageadd: "Ports machinery sprites from /tg/."
```
Comment (SleepyGemmy) [inline html/changelogs/wezzy_machinery_scenery.yml:41]:
```suggestion
  - imageadd: "Ports machinery sprites from /tg/."
  - rscdel: "Removes some unused objects and sprites."
```

**Review by SleepyGemmy — DISMISSED** [review]

**Comment by alsoandanswer** [comment]
!wip

**Comment by alsoandanswer** [comment]
Putting on hold while discussing with other spriters

**Comment by NonQueueingMatt** [comment]
reopen when ready

**Pull request closed without merging**